use-cases/model-fine-tuning-pipeline/data-processing/ray/README.md

```diff
@@ -175,7 +215,8 @@ filter: |-
   severity=ERROR
 labelExtractors:
   gke_node: EXTRACT(labels."compute.googleapis.com/resource_name")
-  ray_woker_node_id: REGEXP_EXTRACT(textPayload, "ray_worker_node_id:(.+) Image")
+  ray_woker_node_id:
```

Comment:
I don't think this is valid, I believe it would need to use a block of flow scalar style.

I thought it was supposed to exclude code blocks.

Reply:
It should have, let me check.

Reply:
Fixed. Disabled embedded language formatting in config because its default value is `auto` (format if Prettier detects the language, and can format it)

Reply:
I've also reformatted all the files after disabling embedded language formatting.